src/NexusMods.Paths/AbsolutePath.cs

```diff
@@ -148,17 +234,17 @@ public void GetFullPath(Span<char> buffer)
 
         Directory.CopyTo(buffer);
 
+        // directory without file name
         if (string.IsNullOrEmpty(FileName)) return;
```

Comment:
We can probably also just check if length == 0, avoiding 1 extra compare and branch; since we know FileName is not null based on nullability.